.github/workflows/ci.yml

```diff
@@ -47,8 +46,25 @@ jobs:
       - name: Build third-party
         run: ./ci/before_script.sh
 
-      - name: Run lint
-        run: ./ci/script.sh
+      - if: ${{ !cancelled() }}
```

Comment:
Does the `if:` automatically mean we're in "expression" mode and don't need the `${{ }}`?  I think you copied that from some of my noobness when I started with GHA.

Reply:
Sometimes. In this case we can't remove `${{ }}`. The rules around if-expression-context are extremely inconsistent.

This is OK:

```yaml
- if: cancelled()
```

This is not OK:

```yaml
- if: !cancelled()
```

I tried to find more information about these "rules" but came up emptyhanded.

Reply:
Maybe `cancelled() == false`?

Reply:
> Maybe `cancelled() == false`?

Sure, that works too. I don't have any strong preference over either syntax.

Reply:
Apparently you have to use quotes if you want to use `!`.